### PR TITLE
Add Pokédex 3D

### PR DESCRIPTION
**Pull request by johnponflanchan**
Adds Pokédex 3D.

*Automatically generated by PR builder*

### CODE REVIEW

File: pokedex-3d/game.dat [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+region = "usa"
+release_date = "2011-06-06"
+
+[[testcases]]
+title = "0004000000032600"
+compatibility = "5"
+date = "2017-07-24"
+version = "HEAD-c6a2e51"
+cpu = "Intel® Core i5-2400"
+gpu = "Intel Iris Pro Graphics 6200"
```
Comment (BreadFish64) [inline pokedex-3d/game.dat]: Are you sure that's the correct GPU? [ARK](http://ark.intel.com/Search?q=Intel%C2%AE%20Iris%E2%84%A2%20Pro%20Graphics%206200) says that Iris Pro 6200 is only available on select 5th gen Broadwell CPUs. And the i5-2400 is a 2nd gen sandy bridge.

**Comment by Schplee** [comment]
Your boxart is poor quality. Did you screenshot the title screen in Citra, as denoted by the [Contributions Guide](https://github.com/citra-emu/citra-games-wiki/blob/master/CONTRIBUTING.md)? 

Please fix this, what @BreadFish64 said, and create a Wiki article for the game.

File: pokedex-3d/game.dat [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+os = "Mac OS X El Capitan"
```
Comment (TheKoopaKingdom) [inline pokedex-3d/game.dat]: You might want to be more specific about which version of El Capitan you're using.

**Comment by Schplee** [comment]
@johnponflanchan Any updates on this PR? Otherwise it will be closed.

**Comment by Schplee** [comment]
PR author looks to have abandoned it. Closing.

**Pull request closed without merging**